src/Containers.jsx

```diff
@@ -113,7 +113,7 @@ const ContainerActions = ({ container, containerDetail, healthcheck, onAddNotifi
 
         client.postContainer(container.isSystem, "pause", container.Id, {})
                 .catch(ex => {
-                    const error = cockpit.format(_("Failed to pause container $0"), container.Names);
+                    const error = cockpit.format(_("Failed to pause container $0"), container.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)